tests/client/coll.rs

```diff
@@ -0,0 +1,50 @@
+use bson::Document;
+use bson::Bson;
+use mongodb::client::wire_protocol::operations::{OpQueryFlags, Message};
+use mongodb::client::MongoClient;
+use mongodb::client::db::Database;
+use mongodb::client::coll::{Collection, FindOptions};
+use std::io::Write;
+use std::net::TcpStream;
+
+//#[test]
+fn find() {
+    let client = MongoClient::with_uri("mongodb://localhost:27017").unwrap();
+    let db = Database::new(&client, "sample", None, None);
```

Comment:
In most drivers, db implements a `collection()` function that makes it so you don't have to do the `Collection::new` below. Makes for less boilerplate. Could we implement that?